src/utils/make_config_hex.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
is it the intention to add this to the `makefile`?  if so when will this be done, in this PR as it matures or another PR?

Reply:
No. See comment in https://github.com/betaflight/betaflight/pull/8583#issuecomment-512069169